functions.py

```diff
@@ -2,3 +2,9 @@ def random_num():
     # returns a random float between 0.0 and 1.0
     from random import random
     return random()
+
+
```

Comment:
Super! Could you please create the documentation of this function

Reply:
will do!